zerver/tests/test_signup.py

```diff
@@ -1501,7 +1501,7 @@ def test_invite_user_as_invalid_type(self) -> None:
         self.login("iago")
         invitee = self.nonreg_email("alice")
         response = self.invite(invitee, ["Denmark"], invite_as=10)
-        self.assert_json_error(response, "Must be invited as an valid type of user")
+        self.assert_json_error(response, "Must be invited as a valid type of user")
```

Comment:
This typo, now fixed, is in various `.po` files, unsure whether I should go hand-edit them, re-run `makemessages` or some equivalent, or just let this get picked up in some future translations run.

Reply:
Nope, it'll just be taken care of automatically the next time we sync translations.